### PR TITLE
(maint) Fix some typos

### PR DESCRIPTION
**Pull request by kylog**

### CODE REVIEW

File: language/expressions.md [hunk 1]
```diff
@@ -1848,11 +1848,11 @@ Definition Expressions
 The Puppet Language has several definition expressions:
 
 * Function definition
-* User defied resource dfinition
+* User defined resource dfinition
```
Comment (hlindberg) [inline language/expressions.md]: You missed the other typo on the same line :-)

Comment (hlindberg) [inline language/expressions.md]: I can fix that without a PR

**Comment by hlindberg** [comment]
dfnitoin is hard to spell - fixed in two more locations (changed on master).

File: language/expressions.md (hunk 1 above)
Comment (kylog) [inline language/expressions.md]: Haha. I have a limit of one typo fix per line.

**Pull request merged**